### PR TITLE
Update touchui-composite-multifield.js

### PR DESCRIPTION
**Pull request by SylvesterAbreu**
This is to address the issue reported here https://github.com/Adobe-Consulting-Services/acs-aem-commons/issues/1970. Credit to https://github.com/awferg

### CODE REVIEW

**Comment by badvision** [comment]
Why not remove the previous line?  Seems we're just setting the same variable twice in a row.  Also the previous line of code is less specific (doesn't indicate 'input' in the query selector).  Is the fact it is being too specific why the bug is occurring?

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js [hunk 1]
```diff
@@ -189,6 +189,10 @@
 
                         _.each(record, function (rValue, rKey) {
                             $field = $($fieldSets[i]).find("[name='./" + rKey + "']").last();
+																												
+																												if ($field.hasClass("coral-RichText-editable")) {
+	                               $field = $($fieldSets[i]).find("input[name='./" + rKey + "']").first();
```
Comment (davidjgonzalez) [inline ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js]: Is this not overwriting the value set on line 191?  What is the rationale for the first vs the last?

@schoudry is this a safe change?

Comment (schoudry) [reply ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js]: i think its not overwriting... indentation is so poor,` if ($field.hasClass("coral-RichText-editable")) {` is not visible

Comment (davidjgonzalez) [reply ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js]: omg....

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js [hunk 2]
```diff
@@ -189,6 +189,10 @@
 
                         _.each(record, function (rValue, rKey) {
                             $field = $($fieldSets[i]).find("[name='./" + rKey + "']").last();
+																												
+																												if ($field.hasClass("coral-RichText-editable")) {
```
Comment (davidjgonzalez) [inline ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js]:
```suggestion																										if ($field.hasClass("coral-RichText-editable")) {
```

**Pull request merged**